app/Services/Utils/DoctrineTransactionService.php

```diff
@@ -59,6 +62,11 @@ public function transaction(Closure $callback, int $isolationLevel = Transaction
         while (!$done and $retry < self::MaxRetries) {
             try {
                 $em  = Registry::getManager($this->manager_name);
+                $event_manager = $em->getEventManager();
+
+                if (!$event_manager->hasListeners(Events::onFlush))
```

Comment:
i do think that this could be done on doctrine config not here

Reply:
done